core/src/main/resources/hudson/model/View/noJob.jelly

```diff
@@ -23,9 +23,8 @@ THE SOFTWARE.
 -->
 
 <?jelly escape-by-default='true'?>
-<j:jelly xmlns:j="jelly:core">
-    ${%description_1}
-    <j:if test="${it.hasPermission(it.CONFIGURE)}">
-        ${%description_2}
-    </j:if>
+<j:jelly xmlns:j="jelly:core" xmlns:l="/lib/layout">
+    <l:notice text="${%description_1}"
+              description="${it.hasPermission(it.CONFIGURE) ? '%description_2' : null}"
```

Comment:
`text` and `description` is a weird pair of attribute titles. Is this only done for backward compatibility? If so, WDYT about deprecating `text` and replacing it with `title`?

Reply:
Only usages are in core so I've renamed them all to `title`, good idea.